src/components/PlaybackControls/PlaybackControls.tsx

```diff
@@ -24,7 +24,7 @@ const PlaybackControls: React.FC = () => {
     }
     if (key.downArrow) {
       setSelectedControlIndex(1);
-      focusNext();
+      focus('search-bar');
```

Comment:
Is there any reasons we should hardcode search-bar as the next focused component? My understanding is that focusNext will switch to whatever focusable component is next in the render tree (and it may not always be search if we decide to move things around).